scripts/create-adaptation-pr.sh

```diff
@@ -111,6 +116,11 @@ if git diff --name-only --diff-filter=U | grep -q .; then
   echo "  1) Open `pwd` in a new terminal and run 'git status'"
   echo "  2) Run 'git add' on the resolved files, but do not commit"
   read -p "  3) Press enter to continue, when you are done"
+else
+  echo
+  echo "### [auto] No further conflicts"
+  echo "### Committing the changes"
+  git commit -m "chore: merge latest changes from 'origin/nightly-testing'"
```

Comment:
This one should only `git add`, because the commit is in fact done after this `if` in the step that prepares a PR and a zulip message.

Reply:
```suggestion
  echo "### Adding the changes"
  git add lean-toolchain lake-manifest.json
```